### PR TITLE
Combine Folders Feature Added

### PR DESCRIPTION
**Pull request by danny007in**
Can combine folders with another folder and lint within as same array result

Normal without combine

`fusv build/scss node_modules/bootstrap/scss`

FdsKcLcIZc

With combine Feature

`fusv build/scss node_modules/bootstrap/scss -c`

UG46BlC9X7

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -42,7 +42,26 @@ function main(args) {
 
         unusedList = unusedList.concat(unusedVars.unused);
         spinner.start();
-    });
+    } else {
+        args.forEach(arg => {
+            const dir = path.resolve(arg);
```
Comment (XhmikosR) [inline cli.js:47]: This whole block needs to be moved to a reusable function

Comment (danny007in) [reply cli.js:47]: I will do

Comment (danny007in) [reply cli.js:30]: Can merge

**Comment by danny007in** [comment]
can merge

**Comment by gentzeng** [comment]
I like this feature. I am currently trying to rebase it with PR #125 to avoid conflicts. Hope you don't mind

**Comment by danny007in** [comment]
> 
> 
> I like this feature. I am currently trying to rebase it with PR #125 to avoid conflicts. Hope you don't mind

No dont do now plz

**Comment by gentzeng** [comment]
Year, that is exactly what I meant. I just fetched your PR #115 to my local repository to update it and to propose it to you. I'm not changing anything here and will notice you, when my proposal is ready for your review

**Comment by danny007in** [comment]
> 
> 
> Year, that is exactly what I meant. I just fetched your PR #115 to my local repository to update it and to propose it to you. I'm not changing anything here and will notice you, when my proposal is ready for your review

I am editing here https://github.com/XhmikosR/find-unused-sass-variables/pull/124

**Comment by gentzeng** [comment]
Allright

**Comment by danny007in** [comment]
does this feature implemented in main branch?

**Comment by XhmikosR** [comment]
I don't think it is. You could try rebasing and maybe @GeoSot or I could have a look later.

**Comment by XhmikosR** [comment]
This needs a proper rebase and make sure that lint and tests pass. Feel free to make a proper, new PR.

**Pull request closed without merging**